src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application "
+      method="post"
+      data-qa="big"
+    >
+      <img
+        class="icon"
+        src="./images/Search.svg"
+        alt="magnifying glass"
+      >
+      <input
+        type="search"
+        class="search-input-big"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application "
+      method="post"
+      data-qa="small"
+    >
+      <img
+        class="icon icon-small"
+        src="./images/Search.svg"
+        alt="magnifying glass"
+      >
+      <input
+        type="search"
+        class="search-input-small"
```

Comment:
You have `search-input-small` and `search-input-big` classes. Let's fix it according to the checklist p.2